releases/release-5.3.0.md

```diff
@@ -0,0 +1,392 @@
+---
+title: TiDB 5.3 Release Notes
+---
+
+# TiDB 5.3 Release Notes
+
+Release date: November 23, 2021
+
+TiDB version: 5.3.0
+
+In v5.3, the key new features or improvements are as follows:
+
++ Introduce temporary tables to simplify your application logic and improve performance
++ Support setting attributes for tables and partitions
++ Support creating users with minimum privilege on TiDB Dashboard to enhance system security
++ Optimize the process of handling timestamps in TiDB to improve the overall performance
++ Enhance the performance of TiDB Data Migration (DM) so that data is migrated from MySQL to TiDB with lower latency
++ Support parallel import in TiDB Lightning to improve the efficiency of full data migration
++ Support saving and restoring the on-site information of a cluster with a single SQL statement, which helps improve the efficiency of troubleshooting execution plan issues
```

Comment:
```suggestion
+ Optimize the timestamp processing flow in TiDB to improve the overall performance
+ Enhance the performance of TiDB Data Migration (DM) so that data is migrated from MySQL to TiDB with lower latency
+ Support parallel import in TiDB Lightning to improve the efficiency of full data migration
+ Support saving and restoring the on-site information of a cluster with a single SQL statement, which helps improve the efficiency of troubleshooting issues about the execution plan
```